themes/default/content/docs/clouds/aws/get-started/_index.md

```diff
@@ -12,8 +12,1418 @@ menu:
 aliases:
 - /docs/quickstart/aws/
 - /docs/get-started/aws/
+- /docs/quickstart/aws/begin/
+- /docs/quickstart/aws/install-pulumi/
+- /docs/quickstart/aws/install-language-runtime/
+- /docs/quickstart/aws/configure/
+- /docs/get-started/aws/install-pulumi/
+- /docs/get-started/aws/install-language-runtime/
+- /docs/get-started/aws/configure/
+- /docs/get-started/aws/begin/
+- /docs/quickstart/aws/create-project/
+- /docs/get-started/aws/create-project/
+- /docs/quickstart/aws/deploy-changes/
+- /docs/get-started/aws/deploy-changes/
+- /docs/quickstart/aws/deploy-stack/
+- /docs/get-started/aws/deploy-stack/
+- /docs/quickstart/aws/destroy-stack/
+- /docs/get-started/aws/destroy-stack/
+- /docs/quickstart/aws/modify-program/
+- /docs/get-started/aws/modify-program/
+- /docs/quickstart/aws/next-steps/
+- /docs/get-started/aws/next-steps/
+- /docs/quickstart/aws/review-project/
+- /docs/get-started/aws/review-project/
 ---
 
-{{< cloud-intro "AWS" >}}
+Pulumi's infrastructure-as-code SDK helps you create, deploy, and manage AWS containers, serverless functions, and infrastructure using familiar programming languages.
 
-{{< get-started-stepper >}}
+This tutorial takes you through the following steps to easily deploy a static website.
+
+## Install Pulumi
+
+{{< install-pulumi >}}
+{{% notes "info" %}}
+All Windows examples in this tutorial assume you are running in PowerShell.
+{{% /notes %}}
+{{< /install-pulumi >}}
+
+### Install language runtime
+
+{{< chooser language "javascript,typescript,python,go,csharp,java,yaml" >}}
```

Comment:
These generally have trailing slashes, and it looks like the page that's been deleted had them too. Why delete them? (This changes how the chooser works -- when they're omitted, they they don't get/set the user's preferred language.)

```suggestion
{{< chooser language "javascript,typescript,python,go,csharp,java,yaml" />}}
```

Reply:
@cnunciato the choosers dont work, if they are self closing like this instead of having the closing chooser after the choosables, it breaks the ones later on the page. happy for you to tell me im wrong, but its the only way i could get them to work.

Reply:
Yeah, that shouldn't happen, as multiple choosers on a single page should work (and works on other pages on the site). Could it be that there are situations where the choosers on a given page don't have the same set of enabled languages? That _could_ happen (but probably shouldn't happen on a getting-started page).

Reply:
i can try adding them back though!

Reply:
im seeing that its saving my selection across pages though 🤔